readthedocs/templates/core/project_details.html

```diff
@@ -38,7 +36,7 @@ <h3>{% trans "Versions" %}</h3>
             {{ version.slug }}
           </a>
         {% endif %}
-          {% if request.user|is_admin:project %}
+          {% if is_project_admin %}
```

Comment:
What's the difference here?

`is_project_admin` is calling `AdminPermission.is_admin` which is exactly the same that `is_admin` filter does.

If we need to use the value multiple times in the template, we can use `{% with request.user|is_admin:project as is_project_admin %}`.

I don't like too much to change the pattern of using `|is_admin` filter here, since it's everywhere and already stablished.

Reply:
@humitos it uses a lot fewer queries. There were over 800 queries on this page in .com because of the permission checks getting really expensive.

Reply:
The `with` statement wouldn't work across templates, so I think this is the correct solution.

Reply:
We would have a big with statement somewhere in a common template, I don't feel that's more readable

Reply:
The code changes to reduce the queries are :100:. I just wanted to try to keep the same pattern we have been using if possible, instead of having a new one only for these files.